docs/src/main/asciidoc/firestore.adoc

```diff
@@ -180,6 +180,37 @@ Read documents are locked until the transaction finishes with a commit or a roll
 If an `Exception` is thrown within a transaction, the rollback operation is executed.
 Otherwise, the commit operation is executed.
 
+===== Declarative Transactions with @Transactional Annotation
+In order to use this feature you would need to do the following.
+
+. Annotate your configuration class with the @EnableTransactionManagement annotation.
+
+. Create a service class:
+
+[source, java, indent=0]
+----
+include::{project-root}/spring-cloud-gcp-data-firestore/src/test/java/org/springframework/cloud/gcp/data/firestore/it/UserService.java[tag=user_service]
+----
+
+[start=3]
+. Make a Spring Bean provider that creates an instance of that class:
```

Comment:
Creating a service and making it a bean seems like normal practice for pretty much anything you do in Spring. Why point it out specifically for `@Transactional`?
I think it's would be more useful to point out the class name of the transaction manager. Sort of what we do for [Spanner](https://github.com/spring-cloud/spring-cloud-gcp/blob/master/docs/src/main/asciidoc/spanner.adoc#declarative-transactions-with-transactional-annotation).
It's OK to keep code snippets in the refdoc, but it's important to point out that it's just one way of doing it.

Reply:
It is done this way to provide the reader with usable code snippets.
I'll add a class name for transaction manager as well.